apps/web/src/components/Follow/FollowListUsers.tsx

```diff
@@ -83,11 +120,16 @@ const StyledList = styled.div`
   display: flex;
   flex-direction: column;
   height: 100%;
-  padding-top: 32px;
+  padding-top: 12px;
 `;
-const StyledListItem = styled.a`
-  padding: 16px;
+
+const StyledListItem = styled.a<{ isMobile: boolean }>`
+  display: flex;
+  padding-top: 16px;
+  padding-bottom: 16px;
   text-decoration: none;
+  gap: ${({ isMobile }) => (isMobile ? '4px' : '72px')};
```

Comment:
lets use breakpoints for mobile styling where possible

gap: 4px
@media only screen and ${breakpoints.desktop} {
    gap: 72px;
  }